docs/core/entity/sensor.md

```diff
@@ -82,6 +83,55 @@ like this month's energy consumption, today's energy production or the yearly gr
 a stock portfolio. The sensor's value when the first statistics is compiled is used as
 the initial zero-point.
 
+#### How to choose `state_class` and `last_reset`
+It's recommended to use state class `total` without `last_reset` whenever possible.
+Setting state class to `total_increasing` or `total` with `last_reset` should only be done if needed.
```

Comment:
Can you explain when it is needed.

Reply:
I mean that state class `total` without `last_reset` is preferred because it has the least surprising behavior, I'll try to rephrase it.

Reply:
Rephrased in 00b558e